modules/flowable-bpmn-model/src/main/java/org/flowable/bpmn/model/AbstractFlowableHttpHandler.java

```diff
@@ -80,7 +80,9 @@ public void setValues(AbstractFlowableHttpHandler otherHandler) {
         super.setValues(otherHandler);
         setImplementation(otherHandler.getImplementation());
         setImplementationType(otherHandler.getImplementationType());
-        Optional.ofNullable(this.scriptInfo).map(ScriptInfo::clone).ifPresent(this::setScriptInfo);
+        if (otherHandler.getScriptInfo() != null) {
```

Comment:
Shouldn't the import for `Optional` be removed too?

Reply:
Fixed. Finally activated to automatically remove unused imports. Hopefully this won't happen this often anymore as of now :)